### DISCUSSION TITLE
Do actions support versioning by the folder name or even in the metadata like other xlets?

### DISCUSSION THREAD

**Discussion by anaximeno** [linuxmint/discussions#329] [Development] [answered]
I don't need it yet, I'm just curious if it is possible to use this already.

**Comment by rcalixte** [comment, marked as answer]
Versioning isn't supported at the moment and it isn't as simple to implement due to how Nemo finds Actions vs how the other xlets are discovered by the system. It would likely take some refactoring in Nemo to support this but that is my speculation. The good news is that most Actions can work around this issue with scripts that check for a specific version if needed. This is already being done in some Actions that check for the specific version of Zenity (version 3 and version 4 have breaking differences).
